git_machete/git_operations.py

```diff
@@ -102,39 +100,39 @@ def get_git_version(self) -> Tuple[int, ...]:
             # We need to cut out the x.y.z part and not just take the result of 'git version' as is,
             # because the version string in certain distributions of git (esp. on OS X) has an extra suffix,
             # which is irrelevant for our purpose (checking whether certain git CLI features are available/bugs are fixed).
-            raw = re.search(r"\d+.\d+.\d+", self.popen_git("version")).group(0)
+            raw = re.search(r"\d+.\d+.\d+", self.get_popen_git("version")).group(0)
             self.git_version = tuple(map(int, raw.split(".")))
         return self.git_version
 
     def get_root_dir(self) -> str:
         if not self.root_dir:
             try:
-                self.root_dir = self.popen_git("rev-parse", "--show-toplevel").strip()
+                self.root_dir = self.get_popen_git("rev-parse", "--show-toplevel").strip()
             except MacheteException:
                 raise MacheteException("Not a git repository")
         return self.root_dir
 
     def __get_git_dir(self) -> str:
         if not self.git_dir:
             try:
-                self.git_dir = self.popen_git("rev-parse", "--git-dir").strip()
+                self.git_dir = self.get_popen_git("rev-parse", "--git-dir").strip()
             except MacheteException:
                 raise MacheteException("Not a git repository")
         return self.git_dir
 
     def get_git_subpath(self, *fragments: str) -> str:
         return os.path.join(self.__get_git_dir(), *fragments)
 
-    def parse_git_timespec_to_unix_timestamp(self, date: str) -> int:
+    def get_git_parse_timespec_to_unix_timestamp(self, date: str) -> int:
```

Comment:
Sounds more natural:
```suggestion
    def get_git_timespec_parsed_to_unix_timestamp(self, date: str) -> int:
```